eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -170,3 +170,8 @@ steps:
         PushArgs: -f
         WorkingDirectory: $(System.DefaultWorkingDirectory)/${{ repo }}
         ScriptDirectory: $(System.DefaultWorkingDirectory)/$(Build.Repository.Name)/${{ parameters.ScriptDirectory }}
+
+- template: /eng/common/pipelines/templates/steps/emit-rate-limit-metrics.yml
+  parameters:
+    GitHubUser: azure-sdk
+    GitHubToken: $(azuresdk-github-pat)
```

Comment:
```suggestion
    GitHubToken: $(azuresdk-github-pat)

```

Reply:
I'll add the line break in another PR when I move the inline script to the github helpers script